PR for SRVLOGIC-353: Document "Managing persistence" section in the OSL docs. (#85633)
openshift/openshift-docs: modules/serverless-logic-profile-persistence-requirements.adoc

// Module included in the following assemblies:
// * serverless-logic/serverless-logic-managing-persistence

:_mod-docs-content-type: REFERENCE
[id="serverless-logic-profile-persistence-requirements_{context}"]
= Profile specific persistence requirements

> The persistence configurations provided for both `SonataFlowPlatform` custom resource (CR) and `SonataFlow` custom resource (CR) apply equally to the `preview` and `gitops` profiles. However, these configurations should not be used with the `dev` profile, as this profile ignores them entirely.

Reviewer: ```suggestion
The persistence configurations provided for both `SonataFlowPlatform` custom resource (CR) and `SonataFlow` CR apply equally to the `preview` and `gitops` profiles. However, you must avoid using these configurations with the `dev` profile, as this profile ignores them entirely.
```

> The primary difference between the `preview` and `gitops` profiles lies in the build process:

Reviewer 2: I think since you're not introducing a format element like a list here, you could just end this sentence with a period. up to you:
```suggestion
The primary difference between the `preview` and `gitops` profiles lies in the build process.
```

When using the `gitops` profile, ensure that the following Quarkus extensions are included in the workflow image during the build process. 

[cols=3*,options="header"]
|===
|groupId
|artifactId
|version

|`io.quarkus`
|`quarkus-agroal`

> |`3.8.4.redhat-00002`

Author: I have added the attribute for this.

|`io.quarkus`
|`quarkus-jdbc-postgresql`

> |`3.8.4.redhat-00002`

Author: Done. I have added the attribute for this.

|`org.kie`
|`kie-addons-quarkus-persistence-jdbc`

> |`9.100.0.redhat-00004`

Author: Done. I have added the attribute for this.

|===

> If you are using the `kogito-swf-builder` to generate your images, you can pass the following build argument to include these extensions:

Author: Done. I have added the attribute for this.

[source,terminal]
----

> $ QUARKUS_EXTENSIONS=io.quarkus:quarkus-agroal:3.8.4.redhat-00002,io.quarkus:quarkus-jdbc-postgresql:3.8.4.redhat-00002,org.kie:kie-addons-quarkus-persistence-jdbc:9.100.0.redhat-00004

Author: Done!

> $ QUARKUS_EXTENSIONS=io.quarkus:quarkus-agroal:3.8.4.redhat-00002,io.quarkus:quarkus-jdbc-postgresql:3.8.4.redhat-00002,org.kie:kie-addons-quarkus-persistence-jdbc:9.100.0.redhat-00004

Reviewer 3: @kaldesai I can see that you added the variables, great work. In the table for example, the values are good. but in this particular part, In the generated document I can see  this:

![image](https://github.com/user-attachments/assets/a8b2fbff-4559-4908-8fe9-63579f037c38)

Not sure if this will be good when we merge, or must be fixed now ¿?
